src/bundle/burn_validation.rs

```diff
@@ -114,9 +121,18 @@ mod tests {
     #[test]
     fn validate_bundle_burn_duplicate_asset() {
         let bundle_burn = vec![
-            get_burn_tuple(b"Asset 1", 10),
-            get_burn_tuple(b"Asset 1", 20),
-            get_burn_tuple(b"Asset 3", 10),
+            get_burn_tuple(
+                &compute_asset_desc_hash(&NonEmpty::from_slice(b"Asset 1").unwrap()),
```

Comment:
can we add a `From<>` trait for `NonEmpty` in our code so we can do `b"Asset 1".into()` ?

Reply:
I think it is not possible because  neither [u8] nor NonEmpty<u8> are defined in the current crate.